modulemd/v2/include/private/modulemd-profile-private.h

```diff
@@ -65,3 +65,15 @@ gboolean
 modulemd_profile_emit_yaml (ModulemdProfile *self,
                             yaml_emitter_t *emitter,
                             GError **error);
+
+/**
+ * modulemd_profile_equals_wrapper:
+ * @a: A void pointer
+ * @b: A void pointer
+ *
+ * Returns: TRUE, if both the pointers are equal. FALSE, otherwise
+ *
+ * Since: 2.3
```

Comment:
This is correctly private API, but note that `Since: 2.3` will no longer be accurate..
```suggestion
 * Since: 2.5
```